.circleci/config.yml

```diff
@@ -36,9 +36,30 @@ jobs:
       - run:
           name: Run build
           command: yarn build
+  publish:
+    docker:
+      - image: circleci/node:11
+    steps:
+      - checkout
+      - run:
+          name: Install Dependencies
+          command: yarn
+      - run:
+          name: Build all code to JavaScript
+          command: yarn build
+      - run:
+          name: Publish
+          command: yarn lerna publish prerelease --yes --dist-tag beta --conventional-commits
```

Comment:
you wanted to change `beta` to `alpha` iirc

Reply:
Done in bf05503